models/cms_hcc/intermediate/cms_hcc__int_members.sql

```diff
@@ -36,6 +37,10 @@ with stg_eligibility as (
             order by enrollment_end_date desc
         ) as row_num /* used to dedupe eligibility */
     from {{ ref('cms_hcc__stg_core__eligibility') }}
+    where
+    /* coverage dates must fall within the collection year */
+    (extract(year from enrollment_start_date) <= {{ collection_year }}
+     and extract(year from enrollment_end_date) >= {{ collection_year }})
```

Comment:
@donaldrauscher this will filter out patients who do not have eligibility in the collection year. Wouldn't we want to keep them and allow them to appear as New Enrollees?

Reply:
Ah, that's a good point.  Having mostly done RA in ACA, I keep forgetting that Medicare RA is prospective.  A member should be considered "New" if they have <12 months of coverage in the collection year, right?

Should we be filtering to members who have any eligibility in either the collection or payment year?

Reply:
> Ah, that's a good point. Having mostly done RA in ACA, I keep forgetting that Medicare RA is prospective. A member should be considered "New" if they have <12 months of coverage in the collection year, right?
> 
> Should we be filtering to members who have any eligibility in either the collection or payment year?

@donaldrauscher Yes, members are considered "New Enrollees" if they have less than 12 months in the collection year. That's a great point about the payment year. It would be weird to show a risk score for someone who doesn't have eligibility in the collection or payment year. I worry about the weird future dates payors like to use, like 2099-12-30. Hopefully, people will clean that before running it through the project since you can't get anything useful from member months with dates like that. I agree we should be filtering to people with eligibility in the collection year or payment year. 

There have been a few changes to the main branch since your PR was open, so you'll need to rebase. Let me know if you need any help!

Reply:
just rebased.

re: future eligibility dates for people with active coverage, agreed.  i think we should assume that this is addressed in eligibility normalization.

made these updates:
- in staging, filter to members with eligibility in collection or payment
- use spans in collection year, apply capping, and determine new vs. continuing (added an aggregation in case you have adjacent spans, wouldn't want to count this as New!)
- then filter to members with eligibility in payment year in `latest_eligibility`.  i think this makes sense.  but might also make sense to have it be any members with payment or collection year?  thoughts?

Reply:
@donaldrauscher these changes look great to me. I just re-ran the CI testing, and all tests passed. One more request: can you update the package to `version: '0.5.10'`? We just released yesterday, so these changes were not able to be included. Thanks for all of your work on this!

https://github.com/tuva-health/the_tuva_project/blob/82c145c0f5efc1b9b7dc72f6743e69cf6b88e2b4/dbt_project.yml#L2

Reply:
Done!  Also updated PR description to reflect latest changes.